app/src/main/java/com/mymeetings/android/utils/ClockUtils.kt

```diff
@@ -2,34 +2,48 @@ package com.mymeetings.android.utils
 
 import android.text.format.DateUtils
 import java.util.*
+import java.util.concurrent.TimeUnit
 
 
 class ClockUtils {
     fun currentTimeMillis() = System.currentTimeMillis()
 
     fun getTimeLeft(timeInMillis : Long): String = DateUtils.getRelativeTimeSpanString(timeInMillis).toString()
 
-    fun getTimeInHoursAndMins(timeInMillis: Long): String {
-        val calendar = Calendar.getInstance()
-        calendar.timeInMillis = timeInMillis
+    fun getTimeInHoursAndMinsWithCurrentDayContext(timeInMillis: Long): String {
+        val todayCalendar = Calendar.getInstance()
+        todayCalendar.set(Calendar.HOUR_OF_DAY, 0)
```

Comment:
`Calendar.HOUR_OF_DAY` gives hours in 24 hour format, which will make 1pm appear as 13pm.  
We have to use `Calendar.HOUR` I suppose.